LICENSE.md

```diff
@@ -0,0 +1,38 @@
+Context data files in this repository are from the [][] spec and
+are licensed by contributors under the [W3C Software and Document License][].
```

Comment:
Is this applicable and is there a W3C spec repo to link to here?

Reply:
It's applicable but we haven't made the spec repo yet. It's on the TODO list. We'll probably need this merged and to be experimenting with it before we get to the spec.